perllib/FixMyStreet/Cobrand/NottinghamshirePolice.pm

```diff
@@ -0,0 +1,63 @@
+=head1 NAME
+
+FixMyStreet::Cobrand::NottinghamshirePolice - code specific to the Nottinghamshire Police cobrand.
+
+=head1 SYNOPSIS
+
+=head1 DESCRIPTION
+
+=cut
+
+package FixMyStreet::Cobrand::NottinghamshirePolice;
+use base 'FixMyStreet::Cobrand::UKCouncils';
+
+use strict;
+use warnings;
+
+use Moo;
+
+=head2 Defaults
+
+=over 4
+
+=cut
+
+sub council_area_id { 2236 }
```

Comment:
There's a hole for Nottingham City in the County area, so to include that I think you'd need:

```suggestion
sub council_area_id { [ 2236, 2565 ] }
```

Reply:
Ah wow, I'd always assumed county areas would be a superset but this makes sense.

Reply:
Ah, is this why the example 'NG2 3DZ' postcode was not returning results for me?

Reply:
No that's something that needs to be be set-up in the server configuration under `example_places` - you can see where I've done that on the `notts-police-staging-setup` branch.